### PR TITLE
e2e tests for Company Audit History

### PR DESCRIPTION
**Pull request by venureddy95**
Acceptance scripts for Companies Audit history

### CODE REVIEW

File: test/acceptance/features/step_definitions/audit-company-step-definitions.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      .navigate()
+      .findCompany(foreignCompanyName)
+    await auditContact
+      .getText('@lastContactFromList', (result) => {
+        companyName = result.value
+      })
+      .click('@lastContactFromList')
+    await auditCompany
+      .editCompanyTwoRecords(description, website)
+  })
```
Comment (yeahfro) [inline test/acceptance/features/step_definitions/audit-company-step-definitions.js]: These bindings can be consolidated by using Regex matching.

File: test/acceptance/features/step_definitions/audit-contacts-step-definitions.js [hunk 2]
```diff
@@ -1,20 +1,129 @@
...
+    await contactCollections
+      .click('@contactsTab')
+    await Contact
+      .getText('@firstCompanyFromList', (result) => {
+        contactName = result.value
+      })
+      .click('@firstCompanyFromList')
+    await auditContact
+      .editContactWithTelephoneAndCountryCode(telephone, countryCode)
   })
```
Comment (yeahfro) [inline test/acceptance/features/step_definitions/audit-contacts-step-definitions.js]: Again these bindings can be consolidated with Regex pattern matching. eg. `I amend ([\d]+) records`

Comment (feedmypixel) [reply test/acceptance/features/step_definitions/audit-contacts-step-definitions.js]: @yeahfro I'm not to sure what you mean here? The above 2 `Givens` have different calls in them. Is combing the work on lines `17-31` and `33-48` what you mean here? I may be missing something

Comment (venureddy95) [reply test/acceptance/features/step_definitions/audit-contacts-step-definitions.js]: @yeahfro : I have addressed your suggestions, made the feature file to use regex and optimized the step definitions and methods involved. Please review.

**Pull request merged**